spec.bs

```diff
@@ -1425,6 +1425,7 @@ To <dfn export>generate a [=/pattern string=]</dfn> from a given [=/part list=]
   1. [=list/For each=] |index| of |index list|:
     1. Let |part| be |part list|[|index|].
     1. Let |last part| be |part list|[|index| - 1] if |index| is greater than 0, otherwise let it be null.
+    1. Let |next part| be |part list|[|index| + 1] if |index| is less than |index list|'s [=list/size] = 1, otherwise let it be null.
```

Comment:
```suggestion
    1. Let |next part| be |part list|[|index| + 1] if |index| is less than |index list|'s [=list/size] - 1, otherwise let it be null.
```